.eslintcache

```diff
@@ -1 +1 @@
sx",[],["37","38"],"/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/reportWebVitals.ts",[],"/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/components/App/App.tsx",["39"],"import React, {useState, useContext, useEffect} from 'react';\nimport {QuestionContext, AllQuestionFormat} from '../../types'\nimport {getQuestions} from '../../apiCalls'\n\n//components\nimport NavBar from '../NavBar/NavBar';\nimport Home from '../Home/Home';\n// import Error from '../Error/Error';\n// import Journey from '../Journey/Journey';\n// import Sourvey from '../Sourvey/Sourvey';\n// import GenerateReport from '../GenerateReport/GenerateReport';\n\n//create interface for context\n\nconst App:React.FC = () =>{\n  const [questions, updateQuestions] = useState<AllQuestionFormat | null>(null);\n  useEffect(() => {\n    getQuestions().then((data) => updateQuestions(data))\n  }, []);\n\n  return (\n    <QuestionContext.Provider value={questions}>\n     \n        <NavBar />\n        <Home />\n        {/* <Sourvey /> */}\n      \n     </QuestionContext.Provider>\n  );\n}\nexport default App;\n\n","/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/components/Sourvey/Sourvey.tsx",[],"/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/components/NavBar/NavBar.tsx",[],"/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/components/Home/Home.tsx",["40","41"],"import React, { useState, useEffect } from 'react';\nimport img from '../../images/home/Big Shoes - Standing Pose.png'\nimport './Home.css'\nconst Home: React.FC = () => {\nconsole.log(img)\n  return (\n    <section className='home-section'>\n      <div className=\"main-container\">\n\n        <div className=\"title-container\">\n          <h1 className='title'>My</h1>\n          <h1 className='title-2'>Dream Home</h1>\n          {/* <h2>Journey Edition</h2> */}\n        </div>\n\n        <div className=\"img-container\">\n          <img src={img} alt=\"\" className='main-image'/>\n        </div>\n\n      </div>\n\n    \n      <div>\n      </div>\n    </section>\n  )\n}\n\nexport default Home;\n","/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/types.tsx",[],"/Users/atos/turing/mod_4/projects/dream_home/FEE_DreamHome/src/apiCalls.tsx",[],{"ruleId":"42","replacedBy":"43"},{"ruleId":"44","replacedBy":"45"},{"ruleId":"46","severity":1,"message":"47","line":1,"column":26,"nodeType":"48","messageId":"49","endLine":1,"endColumn":36},{"ruleId":"46","severity":1,"message":"50","line":1,"column":17,"nodeType":"48","messageId":"49","endLine":1,"endColumn":25},{"ruleId":"46","severity":1,"message":"51","line":1,"column":27,"nodeType":"48","messageId":"49","endLine":1,"endColumn":36},"no-native-reassign",["52"],"no-negated-in-lhs",["53"],"@typescript-eslint/no-unused-vars","'useContext' is defined but never used.","Identifier","unusedVar","'useState' is defined but never used.","'useEffect' is defined but never used.","no-global-assign","no-unsafe-negation"]
```

Comment:
I believe this is an autogenerated cache file from React

Reply:
it is also associated with the Typescript library and started to become an issue with the latest `react-script` upgrade, we are using version 4.0.1 and I tried to downgrade to `4.0.0`  but no luck, I also tried to put the file on the `.gitignore` file but no luck yet. It is just really annoying.